src/App.js

```diff
@@ -33,7 +34,12 @@ function App() {
       <AuthDialog show={show} setShow={setShow} setToken={setToken} setRepository={setRepository} />
       <Stack direction={'row'}>
         <Item>
-          <UploadInterface isAuthorized={isAuthorized} />
+          <UploadInterface
+            isAuthorized={isAuthorized}
+            uploadFile={(file) => {
+              githubUpload(token, repository, file).then(console.log).catch(console.error);
```

Comment:
This isn't really sufficient to indicate success/errors.  We can fix it in a follow-up, but please file an issue to add UI state changes (e.g., "Uploading..." spinner or something)

Reply:
Thank you for your suggestion @humphd, that's a good idea.
I will file another issue and work on it! 🤩